IFC4x3/Properties/a/ACResistance_2$dT1WHiH7Ge2PeqXa73Sp/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ACResistance_2dT1WHiH7Ge2PeqXa73Sp" Name="ACResistance" UniqueId="bf9dd060-46c4-4742-8099-a348641c3733" PrimaryDataType="IfcElectricResistanceMeasure" />
```

Comment:
Would it make sense to generalize as proposed in https://github.com/bSI-InfraRoom/IFC-Specification/pull/161#discussion_r767273174?

Reply:
I think I was referring to "Resistivity".